google_permissions/workgroups.tf

```diff
@@ -1,14 +1,14 @@
 module "admins_workgroup" {
-  source = "../mozilla_workgroup"
+  source = "github.com/mozilla/terraform-modules//mozilla_workgroup?ref=main"
```

Comment:
Is there a release of this we want to pin to?

Reply:
thought about that, too .. v2.6.1 is the most recent one .. I was unsure about adding this "subdependency" here, when we push other modules forward, this is likely going to remain at v2.6.1 in such a case .. :thinking: other modules we define point at the `main` ref, too, e.g. the tenant bootstrapping module does this .. :shrug:

Reply:
I bet this is one of those places that can be forgotten when the main module moves forward .. what would you recommend @bkochendorfer ?

Reply:
e.g. if I merge this, and create a new release of the repo, the pinned version would already be outdated ..